system-test/storage.js

```diff
@@ -696,6 +696,25 @@ describe('storage', function() {
     });
   });
 
+  describe('channels', function() {
```

Comment:
Maybe someday, but I don't think that's meant to turn into a site. Plus, our tests only passing if some dummy server is up is weird. I think we just have to trust the unit tests for these methods :\